auth/index.js

```diff
@@ -26,7 +26,7 @@ module.exports = function (app, hexo) {
       if (req.method === 'POST') {
           req.authenticate(['adminAuth'], function(error, done) {
               if (done) {
-                  res.writeHead(302, { 'Location':  "/admin/" });
+                  res.writeHead(302, { 'Location':  hexo.config.root + "admin/" });
```

Comment:
I think this needs a slash before admin, right? either that or line 7 of `strategy.js` needs to not have the slash